src/sparseml/pytorch/sparsification/quantization/quantize.py

```diff
@@ -232,6 +259,21 @@ def raise_if_torch_quantization_not_available():
         )
 
 
+def _inject_qat_wrapper(
+    root_module: Module,
+    target_submodule_name: str,
+    quantization_scheme: QuantizationScheme,
+):
+    submodule_name_parts = target_submodule_name.split(".")
```

Comment:
could do something like this if you feel like it
```suggestion
    *parent_parts, target_name = target_submodule_name.split(".")
```